MoneySplitter.Win10/ViewModels/AddTransactionViewModel.cs

```diff
@@ -182,7 +182,7 @@ public void RemoveFriendFromCollabarators(int friendId)
 
         public async Task AddTransactionAsync()
         {
-            if(IsSelfCollabarator==true)
+            if (IsSelfCollabarator == true)
```

Comment:
if (IsSelfCollabarator)
jut leave variable, don't compare bool and bool to get bool